packages/react-ui-ag/src/Gmap/markerIcons.js

```diff
@@ -6,26 +6,45 @@ const scaledSize = (x, y, scale) => (
   anchor: scaledPoint(16, 45, scale),
   scaledSize: scaledSize(32, 54, scale),
 })
+
+const redPdpMapBalloon = (scale = 1) => ({
+  url: "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='28' height='40' viewBox='0 0 28 40'%3E%3Cg fill='none' fill-rule='evenodd'%3E%3Cpath fill='%23FF132F' fill-rule='nonzero' d='M14 0C6.27 0 0 6.27 0 14c0 10.5 14 26 14 26s14-15.5 14-26c0-7.73-6.27-14-14-14z'/%3E%3Cellipse cx='14' cy='12.982' fill='%23FFF' rx='5.25' ry='5.263'/%3E%3C/g%3E%3C/svg%3E",
+  anchor: scaledPoint(15, 38, scale),
+})
+
+const redPdpMapBalloonWithBlackDot = (scale = 1) => ({
+  url: "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='28' height='55' viewBox='0 0 28 55'%3E%3Cdefs%3E%3Cfilter id='a' width='100%25' height='100%25' x='0%25' y='0%25' filterUnits='objectBoundingBox'%3E%3CfeGaussianBlur in='SourceGraphic'/%3E%3C/filter%3E%3C/defs%3E%3Cg fill='none' fill-rule='evenodd'%3E%3Ccircle cx='14' cy='48' r='7' fill='%23000' filter='url(%23a)'/%3E%3Cpath fill='%23FF132F' fill-rule='nonzero' d='M14 0C6.27 0 0 6.27 0 14c0 10.5 14 26 14 26s14-15.5 14-26c0-7.73-6.27-14-14-14z'/%3E%3Cellipse cx='14' cy='12.982' fill='%23FFF' rx='5.25' ry='5.263'/%3E%3C/g%3E%3C/svg%3E%0A",
+  anchor: scaledPoint(16, 48, scale),
+})
```

Comment:
You may want a `scaledSize` property for these. Its probably optional since it's unlikely you're using it, but it would be necessary for `scale` to be used properly down the road.

Reply:
Ok, that make sense - I will add in the scaled size property